fuzzing/scripts/build_corpus.py

```diff
@@ -265,7 +265,34 @@ def get_random_operation():
             return operation
 
         def get_random_address_array():
-            return _get_random_array2(self.addresses, 1, self.address_max_count, allow_none=False, unique=True)
+            addresses = _get_random_array2(self.addresses, 1, self.address_max_count, allow_none=False, unique=True)
```

Comment:
I'm not sure I get the point of this logic ?

Side note, should it be at the begining of `get_payload()`? We geenrate `address_max_count` adresses, and always use those adresses.

Reply:
I've added a small commit for namings